<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_open">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>410</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Choisir un fichier FITS</string>
    </property>
   </widget>
   <widget class="QSlider" name="sld_kernel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>460</y>
      <width>611</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="maximum">
     <number>71</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TickPosition::NoTicks</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sld_blur">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>490</y>
      <width>611</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>31</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>460</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kernel Masque</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>500</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Flou Masque</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_display">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>581</width>
      <height>361</height>
     </rect>
    </property>
    <property name="text">
     <string>Pas d'image chargé</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="sld_iter">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>540</y>
      <width>611</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>99</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_iter">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>540</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Iterations</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_kernel">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>460</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="maximum">
     <number>71</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_blur">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>500</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>31</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_iter">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>540</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSlider" name="sld_threshold">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>580</y>
      <width>611</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-30</number>
    </property>
    <property name="maximum">
     <number>-1</number>
    </property>
    <property name="singleStep">
     <number>0</number>
    </property>
    <property name="value">
     <number>-3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_threshold">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>580</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-30</number>
    </property>
    <property name="maximum">
     <number>-1</number>
    </property>
    <property name="value">
     <number>-3</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_threshold">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>580</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Détecte Etoiles</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>904</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sld_kernel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_kernel</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>663</x>
     <y>488</y>
    </hint>
    <hint type="destinationlabel">
     <x>817</x>
     <y>491</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_kernel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sld_kernel</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>804</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>701</x>
     <y>496</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sld_blur</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_blur</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>701</x>
     <y>531</y>
    </hint>
    <hint type="destinationlabel">
     <x>802</x>
     <y>529</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_blur</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sld_blur</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>806</x>
     <y>532</y>
    </hint>
    <hint type="destinationlabel">
     <x>681</x>
     <y>532</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sld_iter</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_iter</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>688</x>
     <y>566</y>
    </hint>
    <hint type="destinationlabel">
     <x>809</x>
     <y>570</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_iter</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sld_iter</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>805</x>
     <y>568</y>
    </hint>
    <hint type="destinationlabel">
     <x>674</x>
     <y>570</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sld_threshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_threshold</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>752</x>
     <y>609</y>
    </hint>
    <hint type="destinationlabel">
     <x>846</x>
     <y>621</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_threshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sld_threshold</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>808</x>
     <y>608</y>
    </hint>
    <hint type="destinationlabel">
     <x>712</x>
     <y>604</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
